rules-java/api/src/main/resources/reports/templates/dependency_report.ftl

```diff
@@ -76,9 +76,9 @@
                                     <dt class="trait">SHA1 hash:</dt>
                                     <dd id="${archiveName}-hash">${sha1}</dd>
                                 </#if>
-                                <#if dependencyProject.name?? && dependencyProject.name != archiveName>
+                                <#if archiveName??>
```

Comment:
Now that I'm seeing the diff, I'm having some doubts. Doesn't this effectively make this whole line redundant? Line 58 already displays the archive name, right?

Reply:
Yep, right now it's a redundant information but line 58 is the `h3` title of the panel so, in the future, it could change the content while this line takes care of showing, no matter what the title shows, the name of the dependency.
For this reason maybe is it worth keeping it?